core_backend/app/dashboard/topic_modeling.py

```diff
@@ -20,6 +21,11 @@
 
 logger = setup_logger()
 
```

Comment:
Why do you have this again since it's in the `config.py` file?

Reply:
Which `config.py` do you mean? I thought I'd replicated the logger flow of other dashboard functionalities but may be wrong